packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -45,6 +45,7 @@ import { defineSystem } from '@etherealengine/ecs/src/SystemFunctions'
 import { InputSystemGroup } from '@etherealengine/ecs/src/SystemGroups'
 import { EngineState } from '@etherealengine/spatial/src/EngineState'
 import { EntityTreeComponent } from '@etherealengine/spatial/src/transform/components/EntityTree'
+import { TransformGizmoTagComponent } from '../../../../editor/src/classes/TransformGizmoControlledComponent'
```

Comment:
this is breaking our import hierarchy

Reply:
hmm, i suppose ill move the gizmo to the sptial package within this PR